src/cred/index.js

```diff
@@ -68,7 +68,7 @@ export function setPhoneNumber(lock, value) {
 }
 
 export function validatePhoneNumber(phoneNumber) {
-  const regExp = /^[0-9]{1,14}$/;
+  const regExp = /^[0-9]([0-9 -])*[0-9]$/;
```

Comment:
You're right. I wanted to reject a phone number ending with a `-` while keeping the regexp straightforward, and that's what I came up with. I think it's safe to assume phone numbers have at least two digits but I'm not sure this is the best validation we can do. Let's wait to have some feedback and then we can improve it.